os/using-environment-variables-in-systemd-units.md

```diff
@@ -0,0 +1,116 @@
+# Using Environment Variables In systemd Units
+
+## Environment directive
+
+Systemd has Environment directive which sets environment variables for executed processes. It takes a space-separated list of variable assignments. This option may be specified more than once in which case all listed variables will be set. If the same variable is set twice, the later setting will override the earlier setting. If the empty string is assigned to this option, the list of environment variables is reset, all prior assignments have no effect.
```

Comment:
This covers the "what" pretty well, but there is no "why". Can we give an example or two about using env vars for configuration? Maybe state that a lot of docker containers use env vars for stuff?